web/src/rest/publish.ts

```diff
@@ -189,6 +189,9 @@ const publishRest = new Vue({
       const { uuid } = asset;
       return `${publishApiRoot}dandisets/${identifier}/versions/${version}/assets/${uuid}/download`;
     },
+    async deleteAsset(identifier: string, version: string, uuid: string): Promise<AxiosResponse> {
+      return client.delete(`${publishApiRoot}dandisets/${identifier}/versions/${version}/assets/${uuid}`);
```

Comment:
```suggestion
      return client.delete(`${publishApiRoot}dandisets/${identifier}/versions/${version}/assets/${uuid}/`);
```
The server should automatically redirect you, but I think it's good practice to have the right URLs wherever possible.